packages/components/src/Components/TagModal/TagModal.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './tagModal.scss';
+import { Modal } from '@patternfly/react-core';
+import {
+    Table,
+    TableHeader,
+    TableBody
+  } from '@patternfly/react-table';
+  
+
+export default class TagModal extends React.Component {
+    constructor(props) {
+        super(props);
+        let rows = []
+        //this bit should be replaced with an API call when the backend is ready
+        if(props.systemName == 'paul.localhost.com') {
```

Comment:
Can you mark this as TODO so we won't forget to remove this and take rows and cells from props?

Reply:
You mean just add TODO: to the comment? If so I just added that

Reply:
Once you are fine with the component and done with testing we should remove this altogether and use rows and cells from props.